### PR TITLE
Fix fixture test

### PR DESCRIPTION
**Pull request by yupwei68**
Fixes #000 

Changes proposed in the pull request:

### CODE REVIEW

File: test/fixture/main.tf [hunk 1]
```diff
@@ -109,6 +112,7 @@ module "debianservers2" {
   vm_os_simple        = var.vm_os_simple_2
   vnet_subnet_id      = azurerm_subnet.subnet2.id
   enable_ssh_key      = true
+  ssh_key             = ""
```
Comment (njuCZ) [inline test/fixture/main.tf:115]: just want to ask why should we set an empty string? I think the module should have the ability to handle it even if not set.

Comment (yupwei68) [reply test/fixture/main.tf:115]: `ssh_key` is not default to empty string. Before we have `extra_ssh_keys`, `ssh_key` is by default to be kept in a path. Then we have `extra_ssh_keys`, we could make `ssh_key` optional by setting it's an empty string.

Comment (njuCZ) [reply test/fixture/main.tf:115]: do you mean that if we don't set this variable for the module, it will have a default non-empty value?

Comment (yupwei68) [reply test/fixture/main.tf:115]: Yes. If we don't set this var, we'll get two ssh_key.

**Review by njuCZ — APPROVED** [review]

**Pull request merged**